React/training/src/app/screens/Main/index.js

```diff
@@ -1,12 +1,21 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import Login from './../Login';
 import Game from './../Game';
+import Topbar from './../../components/Topbar';
 import { connect } from 'react-redux';
+import styles from './styles.module.scss';
 
 class Main extends Component {
   render() {
-    if (this.props.token) {
-      return <Game />;
+    if (this.props.token || localStorage.getItem("token")) {
```

Comment:
extract `localStorage.getItem("token")` to a global function in an `utils.js` to get this.